bin/gce/google_cloud_docker_run.sh

```diff
@@ -0,0 +1,37 @@
+export SPARK_HOME=/usr/lib/spark
+export SPARK_CONF_DIR=/usr/lib/spark/conf
```

Comment:
This file is specifically for mango notebook. we can pass a user variable here that states whether to run notebook or browser based on the --entrypoint flag

Reply:
separate into run-notebook.sh and run-browser.sh